fs/glob.ts

```diff
@@ -86,9 +82,29 @@ export function isGlob(str: string): boolean {
 
 export interface ExpandGlobOptions extends GlobOptions {
   root?: string;
+  exclude?: string[];
   includeDirs?: boolean;
 }
 
+const SEP_PATTERN = isWindows ? /[\\/]+/ : /\/+/;
+
+interface SplitPath {
+  segments: string[];
+  isAbsolute: boolean;
+  hasTrailingSep: boolean;
+}
+
+// TODO: Maybe make this public somewhere if it can be fixed for Windows.
+function split(path: string): SplitPath {
+  return {
+    segments: path
+      .replace(new RegExp(`^${SEP_PATTERN.source}|${SEP_PATTERN.source}$`), "")
```

Comment:
if you want to replace both leading and trailing separators, you probably need "g" modifier in regexp.

```
> "/aaa/".replace(/^\/|\/$/, '')
aaa/
> "/aaa/".replace(/^\/|\/$/g, '')
aaa
```

Reply:
👍